lib/sqlalchemy/sql/functions.py

```diff
@@ -49,7 +55,48 @@ def register_function(identifier, fn, package="_default"):
 
     """
     reg = _registry[package]
```

Comment:
**mike bayer** (zzzeek) wrote:

this looks GREAT, way simpler than the previous version and won't break under concurrency.